zip-0317b.rst

```diff
@@ -0,0 +1,161 @@
+::
+
+  ZIP: 0317b
```

Comment:
The ZIP editors discussed two other alternatives:
2. adding separate sections to the existing ZIP-317 which say what changes need to be made for ZSAs, with a deployment section that says they are made in whichever network upgrade ZSAs are scheduled in
3. adding separate sections to the relevant ZSA ZIP which describe the changes that need to be made to ZIP-317 due to that ZIP

If the changes are small, we have a preference for option 3, because all the changes for the ZSA ZIPs are together in the one place.

Otherwise, option 2 would be better.

Reply:
Update: We have added separate sections to ZIP 317 to say the changes that occur for ZSAs, which corresponds to option 2 above. These changes are in [PR#667](https://github.com/zcash/zips/pull/667).